### ISSUE TITLE
over order_by doesn't allow multiple clauses

### ISSUE THREAD

**Issue by InsidersByte** [sqlalchemy/sqlalchemy2-stubs#156]
**Describe the bug**
The typing of over order_by parameter doesn't allow passing multiple 'order by' clauses.

**Expected behavior**
To be able to pass multiple order bys

**To Reproduce**
```py
from sqlalchemy import Column, Integer, create_engine, func
from sqlalchemy.orm import Session, declarative_base, sessionmaker

Base = declarative_base()


class Foo(Base):
    __tablename__ = "foo"

    id = Column(Integer(), primary_key=True)
    a = Column(Integer())
    b = Column(Integer())


engine = create_engine("")
SessionFactory = sessionmaker(bind=engine)
session: Session = SessionFactory()

session.query(
    func.row_number().over(
        order_by=[
            Foo.a.desc(),
            Foo.b.desc(),
        ],
    )
)
```

**Error**

```
Argument "order_by" to "over" of "FunctionElement" has incompatible type "List[ColumnElement[NullType]]"; expected "Optional[ClauseElement]"
```

**Versions.**
 - OS:
 - Python: 3.8.6
 - SQLAlchemy: 1.4.23
 - mypy:
 - SQLAlchemy2-stubs: 0.0.2a9

**Comment by CaselIT** [comment]
thanks for the report.

**Comment by CaselIT** [comment]
released as v0.0.2a10
